### ISSUE TITLE
Add "Reload button" to EASE error page to retry connecting to site

### ISSUE THREAD

**Issue by Giltyhub2** [EFForg/https-everywhere#18140]
[//]: # (Thank you for reporting an issue to HTTPS Everywhere.)
[//]: # (We welcome input from users on improving this project.)
[//]: #
[//]: # (Please help us by following this issue template.)
[//]: # (You can delete all blank lines and all lines starting)
[//]: # (with the comment marker, such as this one.)

Type: feature request

[//]: # (Include any other relevant information below. Thank you again for)
[//]: # (helping to improve HTTPS Everywhere.)

this will prove to be very practical since currently one has to go to the url bar and remove the entire `moz-extension://...` + clicking on enter

**Comment by pipboy96** [comment]
Do you mean you want it to happen when EASE is disabled but some cancelled pages are still opened?

**Comment by Giltyhub2** [comment]
@pipboy96 no, I want it to happen when EASE is enabled, since sometimes your connection is lost and HTTPS-E (erroneously) displays that page when accessing some site so you want to retry connecting to it
